Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1558,6 +1862,8 @@ static void loadPrimitiveGameThreadPart(
   pMaterial->SetFlags(
       RF_Transient | RF_DuplicateTransient | RF_TextExportTransient);
 
+  // TODO: why were these called twice, is it okay to remove?
```

Comment:
Could we check whether material layers are even used and do it the old way if there are no layers?

Reply:
The only concern I had was vaguely that it would pollute some "global namespace" between all the layers, which might particularly effect the metadata layer that can have all sorts of custom names. Admittedly it's contrived, but it brought my attention to these lines. If that's not a worry I'll leave it as is, uncommenting it for now.

Reply:
I don't think setting global parameters will affect layer parameters at all, even if they share a name. But setting this parameter values that probably don't even exist is a bit wasteful, and it does prevent users from using the same names for global (non-layer) parameters. I wrote #814 for this.